static/css/components/header-legacy.less

```diff
@@ -0,0 +1,151 @@
+@import (less) "less/z-index.less";
+
+div#header {
+  width: 960px;
+  &Search {
+    width: 316px;
+    padding: 9px;
+    outline: none;
+  }
+  &Nav {
+    position: relative;
+    float: left;
+    width: 370px;
+    height: 150px;
+    li.first {
+      margin-right: 279px !important;
+    }
+    li.second {
+      margin-right: 99px !important;
+    }
+    ul.links {
+      position: absolute;
+      left: 1px;
+      bottom: 0;
+      clear: left;
+      float: left;
+      width: 369px;
+      li {
+        display: block;
+        float: left;
+        width: 90px;
+        color: @black;
+        margin-right: 10px;
+        a,
+        span {
+          display: block;
+          width: 80px;
+          height: 22px;
+          padding: 10px 0 0 10px;
+          font-family: "Lucida Grande", Verdana, Geneva, Helvetica, Arial, sans-serif;
+          font-size: .6875em;
```

Comment:
How did we come up with this font-size? inspection of computed values?

Reply:
I have no idea :)
This change is just moving CSS I haven't had time to inspect it closely.
I've pulled this patch out into a new pull request:
https://github.com/internetarchive/openlibrary/pull/1331